### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
I have to figure out how not to get 2 nightly builds :P

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ php:
     - 7.2
     - 7.3
     - 7.4
+    - nightly
```
Comment (alcaeus) [inline .travis.yml:8]: Remove this nightly. This will run a nightly build in addition to the job below

Comment (greg0ire) [reply .travis.yml:8]: Ok, not sure why we don't have 2 jobs on doctrine/annotations then

Comment (alcaeus) [reply .travis.yml]: Because I removed it there

Comment (greg0ire) [reply .travis.yml]: Ah that makes sense :joy:

**Pull request merged**